ui/cap-react/src/antd/admin/utils/schemaSettings.js

```diff
@@ -67,6 +67,7 @@ export const configSchema = {
         type: "string",
         title: "Schema ID",
         description: "Unique ID of the schema",
+        readOnly: true
```

Comment:
For new schemas, if we don't allow this field to be modified here, then it would need to be filled in by default with the id provided when creating it in the admin index